src/clj/fluree/db/query/fql/syntax.cljc

```diff
@@ -131,12 +131,12 @@
     ::select          [:orn {:error/message "Select must be a valid selector or vector of selectors"}
                        [:selector ::selector]
                        [:collection [:sequential ::selector]]]
-    ::direction       [:orn {:error/message "Direction must be ASC or DESC"}
+    ::direction       [:orn {:error/message "Direction must be \"asc\" or \"desc\""}
                        [:asc [:fn asc?]]
                        [:desc [:fn desc?]]]
     ::ordering        [:orn {:error/message "Ordering must be a var or two-tuple formatted ['ASC' or 'DESC', var]"}
                        [:scalar ::var]
-                       [:vector [:and list?
+                       [:vector [:and sequential?
```

Comment:
I think the ordering must be an explicit list to differentiate it from our syntactic usage of vectors for things like groups of variables.